x/dex/spec/README.md

```diff
@@ -1,21 +1,46 @@
 ## Abstract
-
-TODO: Populate dex README Contents below.
-
-## Contents
+`dex` module is responsible for matching orders for registered contracts.
 
 ## Concepts
-
-## State
-
-## Messages
-
-## Events
+### Frequent Batch Auction
```

Comment:
can we add ordering of how orders are processed, similar to whitepaper (cancels first, then add limit orders, then market orders, then fill limit orders)